lib/modules/apostrophe-events-pages/index.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     { name: 'month' },
     { name: 'day' }
   ],
+  perPage: 1,
```

Comment:
I'm a little confused by this. Might be fine, but it's not clear why we're setting `perPage` to one here.

Reply:
this must have been some old experiment. Gone now